cli.js

```diff
@@ -63,48 +102,59 @@ function createDeposit({ nullifier, secret }) {
   return deposit
 }
 
+async function backupNote({ currency, amount, netId, note, noteString }) {
+  try {
+    await fs.writeFileSync(`./backup-tornado-${currency}-${amount}-${netId}-${note.slice(0, 10)}.txt`, noteString, 'utf8');
+    console.log("Backed up deposit note as",fileName)
```

Comment:
Now the script will generate the same deposit backup file like the website so that users could prevent losing funds by closing the command terminal.